### PR TITLE
Dummy monsters behavior

### PR DESCRIPTION
**Pull request by Jedzkie**
- Can't be knock back in kRO
- Corrected dummy names.

### CODE REVIEW

File: src/map/skill.c [hunk 1]
```diff
@@ -2089,7 +2089,7 @@ int skill_blown(struct block_list* src, struct block_list* target, int count, in
 		case BL_MOB:
 		{
 			const struct mob_data *md = BL_UCCAST(BL_MOB, target);
-			if (md->class_ == MOBID_EMPELIUM)
+			if (md->class_ == MOBID_EMPELIUM || md->class_ >= MOBID_DUMMY_10 && md->class_ <= MOBID_DUMMY_10_FIRE)
```
Comment (dastgir) [inline src/map/skill.c]: Missing parenthesis

Comment (Emistry) [inline src/map/skill.c]: about this ... why not set in mob_db.conf ?? i remember there was a settings that disable monsters from knocked back by players.

Comment (dastgir) [inline src/map/skill.c]: There isn't any, though we can add one and remove hard coded stuff

**Comment by Emistry** [comment]
@dastgir  this ?
capture

**Comment by dastgir** [comment]
```

MD_CANMOVE            | 0x0001 |      1
MD_LOOTER             | 0x0002 |      2
MD_AGGRESSIVE         | 0x0004 |      4
MD_ASSIST             | 0x0008 |      8
MD_CASTSENSOR_IDLE    | 0x0010 |     16
MD_BOSS               | 0x0020 |     32
MD_PLANT              | 0x0040 |     64
MD_CANATTACK          | 0x0080 |    128
MD_DETECTOR           | 0x0100 |    256
MD_CASTSENSOR_CHASE   | 0x0200 |    512
MD_CHANGECHASE        | 0x0400 |   1024
MD_ANGRY              | 0x0800 |   2048
MD_CHANGETARGET_MELEE | 0x1000 |   4096
MD_CHANGETARGET_CHASE | 0x2000 |   8192
MD_TARGETWEAK         | 0x4000 |  16384
```
That's our, surely that is rA Custom, but as I was telling above, its good to have Knockback in mode

**Comment by dastgir** [comment]
@Emistry #1093  :)

**Comment by dastgir** [comment]
to be merged after #1093 has been approved/rejected

**Comment by Jedzkie** [comment]
@dastgir

**Comment by dastgir** [comment]
Branch has conflicts, update it with Hercules last version

**Pull request closed without merging**